sysutils/pfSense-pkg-nut/files/etc/inc/priv/nut.priv.inc

```diff
@@ -15,7 +17,7 @@
 	   notice, this list of conditions and the following disclaimer in the
 	   documentation and/or other materials provided with the distribution.
 
-	THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+	THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
```

Comment:
Please do not change license text. I'll import it to unbreak the ports build and will revert it after. Thanks